### ISSUE TITLE
First time logging in fails.

### ISSUE THREAD

**Issue by MatthewSH** [navarr/MinecraftAPI#3]
Consistently when people try to log in, the first time they try it comes back with failed credentials. Not sure if anyone else if having this issue.

**Comment by navarr** [comment]
You're getting a BadLoginException specifically, every first try?

**Comment by MatthewSH** [comment]
yeah, in less you've logged in previously in that...probably 4-10 hours? It's very strange. 

I'm using Laravel 5.3. [Here](https://github.com/halfpetal/CapesAPI/blob/master/app/Http/Controllers/Mojang/AuthController.php#L54) is the line it's used on. I'm handling everything correctly; but it's still strange. The only thing I can think of is that it doesn't respond "in time" so it just throws it to the first catch? I'm not even sure.

However, could it be that Mojang is blocking requests? The error code `ForbiddenOperationException` is not very descriptive. It's associated with bad login credentials, however...could it be something more @navarr?
